Makefile

```diff
@@ -0,0 +1,5 @@
+cs:
+	php-cs-fixer fix --verbose
+
+test:
+	./vendor/bin/phpunit --coverage-text
```

Comment:
I would not run the test coverage by default. Collecting coverage slows down the testsuite, so you don't need to do it all the time locally